tests/v2/test_message_store_sqlite.nim

```diff
@@ -25,13 +25,16 @@ const
 proc newTestDatabase(): SqliteDatabase =
   SqliteDatabase.init("", inMemory = true).tryGet()
 
+proc now(): Timestamp =
+  getNanosecondTime(getTime().toUnixFloat())
+
 proc getTestTimestamp(offset=0): Timestamp = 
-  Timestamp(getNanosecondTime(epochTime()))
+  Timestamp(getNanosecondTime(getTime().toUnixFloat() + offset.float))
 
 proc fakeWakuMessage(
   payload = "TEST-PAYLOAD",
   contentTopic = DefaultContentTopic, 
-  ts = getNanosecondTime(epochTime())
+  ts = now()
```

Comment:
From the documentation, `now` does not return the epoch time, but we need epoch time for the sender timestamp.

Reply:
If you check the defined functions in this test suite module, there's a method called `now` added to improve readability readability:

```nim
proc now(): Timestamp =
  getNanosecondTime(getTime().toUnixFloat())
```